tests/unit/js/test.search.js

```diff
@@ -0,0 +1,165 @@
+import jquery from 'jquery';
+import { more, less } from '../../../openlibrary/plugins/openlibrary/js/search.js';
+
+/** Creates a dummy HTML document with a list of 'facetEntry' element and a
+ * 'facetMoreLess' section.
+ */
+function setDocument(totalFacet = 2, visibleFacet = 2, minVisibleFacet = 2) {
+    document.body.innerHTML = `
```

Comment:
Could `document.createElement` be used here?

Reply:
Not sure what you mean here. Anyway, I've updated